apps/condo/domains/common/schema/fields.js

```diff
@@ -1,14 +1,17 @@
 const { Relationship, Select, Integer, Text } = require('@keystonejs/fields')
 const { Decimal } = require('@keystonejs/fields')
 
+const { GQLError } = require('@open-condo/keystone/errors')
 const { Json, SignedDecimal } = require('@open-condo/keystone/fields')
 
 const { ISO_CODES } = require('@condo/domains/common/constants/currencies')
-const { PHONE_WRONG_FORMAT_ERROR, JSON_UNKNOWN_VERSION_ERROR, REQUIRED_NO_VALUE_ERROR, JSON_EXPECT_OBJECT_ERROR } = require('@condo/domains/common/constants/errors')
+const { PHONE_WRONG_FORMAT_ERROR, JSON_UNKNOWN_VERSION_ERROR, REQUIRED_NO_VALUE_ERROR, JSON_EXPECT_OBJECT_ERROR, COMMON_ERRORS } = require('@condo/domains/common/constants/errors')
 const { normalizePhone } = require('@condo/domains/common/utils/phone')
 const { UNIT_TYPES, FLAT_UNIT_TYPE } = require('@condo/domains/property/constants/common')
 const { ADDRESS_META_FIELD_GRAPHQL_TYPES, ADDRESS_META_SUBFIELDS_QUERY_LIST } = require('@condo/domains/property/schema/fields/AddressMetaField')
 
+const { CALL_RECORD_ERRORS } = require('../../ticket/constants/errors')
```

Comment:
You can’t import ticket from common domain

Reply:
It's unused import, removed it